package.json

```diff
@@ -58,27 +57,36 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "postinstall": "rm -f node_modules/web3/index.d.ts"
+    "postinstall": "rm -f node_modules/web3/index.d.ts",
+    "copy_truffle_files": "copyfiles './contracts/**/*' './migrations/**/*' ./build/dist",
+    "truffle:compile": "(run-s copy_truffle_files && cd build/dist && ../../node_modules/.bin/truffle compile)",
+    "truffle:migrate": "(cd build/dist && ../../node_modules/.bin/truffle migrate)",
+    "truffle:test": "(cd build/dist && ../../scripts/run-with-ganache.sh"
```

Comment:
The compiled tests currently don't run. Maybe the also have to be written in bad-style typescript?

Reply:
Ok, they now run, thanks to separate configuration for truffle-related files. They fail miserably, however.